src/constants.ts

```diff
@@ -1,5 +1,4 @@
 import path from 'node:path';
 
-export const VIDEO_PATH = path.join(process.cwd(), '/video');
-export const CONFIG_PATH = path.join(VIDEO_PATH, '/assets.json');
-export const FILES_PATH = path.join(VIDEO_PATH, '/files');
+export const VIDEOS_PATH = path.join(process.cwd(), '/videos');
+export const CONFIG_PATH = path.join(VIDEOS_PATH, '/assets.json');
```

Comment:
I don't believe this is used anywhere, this is a relic from before we moved to each asset having its own json file.